generate_alias.py

```diff
@@ -0,0 +1,17 @@
+import hashlib
+import string
+import datetime
+
+
+ALPHANUMERIC_CHARS = string.ascii_letters + string.digits
+
+def generate_alias(url):
+  
```

Comment:
add a docstring in here for what this function does
```suggestion
  """
  generates a random 5 character string with alphanumeric characters,
  for example <paste example here>
  """
```

Reply:
done